net/newsraft/Portfile

```diff
@@ -5,20 +5,20 @@ PortGroup           legacysupport 1.1
 PortGroup           makefile 1.0
 PortGroup           codeberg 1.0
 
-codeberg.setup      newsraft newsraft 0.23 newsraft-
-revision            1
+codeberg.setup      newsraft newsraft 0.24 newsraft-
+revision            0
 
 categories          net
 license             ISC
 maintainers         {@sikmir disroot.org:sikmir} openmaintainer
 description         Feed reader for terminal
 long_description    {*}${description}
 
-checksums           rmd160  cca3f1900994a43caad4c8454c2d97061ad1754e \
-                    sha256  22c835f56ff84a4aadc86d9e56a5d8e531cc966ff6a1404f0b3f8f1a1a0655dc \
-                    size    139740
+checksums           rmd160  be517ee826b23572d53d91404c362d5ac206c35e \
+                    sha256  1bc072350cee8960880cef0f749922c9474d91c9df467a0645b1632fbddb48f5 \
+                    size    144186
 
-depends_build       port:pkgconfig
+depends_build       bin:pkg-config:pkgconfig
```

Comment:
```suggestion
depends_build       path:bin/pkg-config:pkgconfig
```

Reply:
@ryandesign Just for informational purpose, is there a reason why `bin:binary:port` syntax does not work here? It is used for cmake elsewhere, for example.

Reply:
`bin:` and `lib:` dependency styles allow files outside of the MacPorts prefix to satisfy the dependency. They check all bin and lib dirs, respectively, not just MacPorts-provided ones. In other words, if the user had /usr/local/bin/pkg-config or /usr/local/bin/cmake, that would satisfy the dependency. This would be undesirable for pkgconfig, cmake, and most other cases since we want known-good MacPorts versions (possibly with MacPorts customizations and path selections) to be used. Allowing non-MacPorts versions is only desirable for common utilities like awk, grep, perl, or sed if one doesn't care which version is used.

`path:` style ensures that a file at the given path is used for dependency resolution. Non-absolute paths are implicitly prepended with `${prefix}/`.

The reason why I've been switching pkgconfig from `port:` to `path:` style whenever I remember to do so is that we also have the pkgconf port which provides an alternative implementation of the pkgconfig concept.

Reply:
Thank you! Yes, I recently discovered `pkgconf` port and understood why path-style made sense, but did not know the difference you described above.